### PR TITLE
Makes directional windows only block air if anchored

### PR DESCRIPTION
**Pull request by SamHPurp**
Porting this fix https://github.com/tgstation/tgstation/pull/31695

🆑 Purpose
fix: Directional windows now only block if they are anchored.
/ 🆑

### CODE REVIEW

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -207,6 +207,7 @@
 				to_chat(user, "<span class='notice'>You begin to [anchored ? "unscrew the window from":"screw the window to"] the floor...</span>")
 				if(do_after(user, decon_speed*I.toolspeed, target = src, extra_checks = CALLBACK(src, .proc/check_anchored, anchored)))
 					anchored = !anchored
+					air_update_turf(TRUE)
```
Comment (FlattestGuitar) [inline code/game/objects/structures/window.dm]: I can't figure out why you only update air on the non-reinforced window after screwdrivering

File: code/game/objects/structures/window.dm [hunk 2]
```diff
@@ -329,7 +330,6 @@
 		return FALSE
 
 	setDir(target_dir)
-	air_update_turf(1)
```
Comment (FlattestGuitar) [inline code/game/objects/structures/window.dm]: You removed it from the counter-clockwise spin, but not the clockwise spin

**Comment by FlattestGuitar** [comment]
ffs tg

**Comment by SamHPurp** [comment]
@FlattestGuitar

**Pull request merged**